fitlins/workflows/base.py

```diff
@@ -2,18 +2,39 @@
 import warnings
 
 
-def init_fitlins_wf(database_path, out_dir, analysis_level, space,
-                    desc=None, model=None, participants=None,
-                    smoothing=None, drop_missing=False,
-                    estimator=None, errorts=False, drift_model=None,
-                    base_dir=None, name='fitlins_wf'):
+def init_fitlins_wf(
+    database_path,
+    out_dir,
+    layout,
```

Comment:
This is new. The test is failing because we're not passing it in the test. I think with `database_path`, we shouldn't need it, though.